web/pages/projects.md

```diff
@@ -83,3 +83,4 @@ let us know on our [socials](https://kframework.org/#support), and we will featu
 - [Elrond (2020-2021)](https://github.com/runtimeverification/elrond-semantics)
 - [P4 (2016-2021)](https://github.com/kframework/p4-semantics)
 - [K-Javalette (2022)](https://github.com/bbyalcinkaya/k-javalette)
+- [K-ST(2023)](https://github.com/wkyml/K-ST)
```

Comment:
```suggestion
- [K-ST (2023)](https://github.com/wkyml/K-ST)
```

Reply:
We have made the necessary revisions. Thank you very much!